invenio_rdm_records/services/schemas/parent/access.py

```diff
@@ -40,6 +40,20 @@ class Grant(Schema):
     permission = fields.String(required=True)
     subject = fields.Nested(GrantSubject, required=True)
     origin = fields.String(required=False)
+    message = SanitizedUnicode()
+    notify = fields.Bool()
+
+
+class Grants(Schema):
+    """Grants Schema."""
+
+    grants = fields.List(
```

Comment:
I don't think we need an extra `grants` container field. It could be just a list of `Grants`. Grants are also included in the parent access [schema](https://github.com/anikachurilova/invenio-rdm-records/blob/b4990bc4c27454c25ab3999275740d799a2924cf/invenio_rdm_records/services/schemas/parent/access.py#L102), how does it look like to return grants in the serializer? i.e. do we have 
```json
"grants": {
  "grants": {...} 
}

```

Reply:
this new schema is not related to parent schema. Parent schema will still return `"grants": []`